### DISCUSSION TITLE
get_parabolic_sa - how to check on the chart

### DISCUSSION THREAD

**Discussion by KrisSzyp** [DaveSkender/Stock.Indicators#1212] [Help and support]
I'm a beginner, sorry for my stupid question.
I use this indicator on pocketoption but it is different from the one they use. How or where can I check your indicator to see what it actually shows at a given moment? Is there any conversion to MT4. Please help.

**Comment by DaveSkender** [comment]
I don’t know a lot about MT4 or _pocketoption_, however our indicator output is indexed by date, checkout the [find()](https://python.stockindicators.dev/utilities/#find-indicator-result-by-date) utility.  For general information to help troubleshoot matching, see:

- #801
